### ISSUE TITLE
IE 10/11 Flex Basis Bug

### ISSUE THREAD

**Issue by erwstout** [erwstout/ginger#26]
Ginger has the following issue as pointed out by [Phillip Walton](https://github.com/philipwalton/flexbugs)

> An explicit flex-basis value (i.e., any value other than auto) is supposed to act just like width or height. It determines the initial size of a flex item and then the other flexibility properties allow it to grow or shrink accordingly.
> 
> IE 10-11 always assume a content box model when using flex-basis to determine a flex item's size, even if that item is set to box-sizing:border-box. Demo 7.1.a shows that an item with a flex-basis value of 100% will overflow its container by the amount of its border plus its padding.

The fix should look something like this:

``` css
.col-4{
  width: 33.33%;
  flex-basis: auto;
}
```

**Comment by erwstout** [comment]
``` scss
// Ginger IE 10/11 Fixes

@media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
  @for $i from 1 through 12 {
    $width: 8.333333% * $i;
    .col-#{$i} {
        width: $width;
        flex-basis: auto;
    }
  }
}
```

**Comment by erwstout** [comment]
Fixed in the latest release, `1.1.3`
